langs/vlang.yaml

```diff
@@ -0,0 +1,29 @@
+id: "v"
+aliases:
+  - "vlang"
+name: "V"
+
+install:
+  manual: |
+    git clone https://github.com/vlang/v
+    pushd v
+    make
+    popd
```

Comment:
Is this compiling a single binary? If so, we should copy it out into `${pkg}/usr/local/bin`. Otherwise, we can put the necessary assets under `${pkg}/opt/v` and then just link the entry point into `${pkg}/usr/local/bin`.

Reply:
[I added the cp](https://github.com/raxod502/riju/pull/137/files#diff-624193f7ebbf28b4f735efdc6458836c96c84e808406009559c1146520f2b273R31), when you get the chance to take a look.